src/common/components/issue-filing-button.tsx

```diff
@@ -4,21 +4,21 @@ import { autobind } from '@uifabric/utilities';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
 import * as React from 'react';
 
-import { BugFilingServiceProvider } from '../../bug-filing/bug-filing-service-provider';
-import { BugFilingService } from '../../bug-filing/types/bug-filing-service';
 import { IssueFilingDialogDeps } from '../../DetailsView/components/issue-filing-dialog';
+import { IssueFilingServiceProvider } from '../../issue-filing/issue-filing-service-provider';
+import { IssueFilingService } from '../../issue-filing/types/issue-filing-service';
 import { EnvironmentInfo, EnvironmentInfoProvider } from '../environment-info-provider';
 import { LadyBugSolidIcon } from '../icons/lady-bug-solid-icon';
 import { BugActionMessageCreator } from '../message-creators/bug-action-message-creator';
```

Comment:
Consider renaming `BugActionMessageCreator` to `IssueFilingActionMessageCreator`?

Reply:
Will do in another PR due to the size of this PR.